src/Sylius/Behat/Resources/config/suites/ui/cmf/static_content.yml

```diff
@@ -0,0 +1,19 @@
+# This file is part of the Sylius package.
+# (c) Paweł Jędrzejewski
+
+default:
+    suites:
+        ui_static_content:
+            contexts_as_services:
+                - sylius.behat.context.hook.doctrine_orm
+                - sylius.behat.context.hook.phpcr
+
+                - sylius.behat.context.transform.shared_storage
+
+                - sylius.behat.context.setup.channel
+                - sylius.behat.context.setup.static_content
+
+                - sylius.behat.context.ui.shop.static_content
+
```

Comment:
Yup. Indention separates them enough. We don't have such a separation in any other suite.